proxy-router/internal/aiengine/prodia_v2.go

```diff
@@ -0,0 +1,108 @@
+package aiengine
+
+import (
+	"bytes"
+	"context"
+	b64 "encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	c "github.com/MorpheusAIs/Morpheus-Lumerin-Node/proxy-router/internal"
+	gcs "github.com/MorpheusAIs/Morpheus-Lumerin-Node/proxy-router/internal/chatstorage/genericchatstorage"
+	"github.com/MorpheusAIs/Morpheus-Lumerin-Node/proxy-router/internal/lib"
+	"github.com/sashabaranov/go-openai"
+)
+
+const API_TYPE_PRODIA_V2 = "prodia-v2"
+const PRODIA_V2_DEFAULT_BASE_URL = "https://inference.prodia.com/v2"
+
+var (
+	ErrCapacity               = errors.New("unable to schedule job with current token")
```

Comment:
@alex-sandrk do these error messages get surfaced back to the ui-desktop if triggered or just in the logs?

Reply:
it will if you chat with local model as a provider. It's provider side error, so the client of remote session will not see it.